### PR TITLE
add kafka health check

### PR DESCRIPTION
**Pull request by jisookim0513**
Addresses issue #57.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: app/controllers/KafkaHealthCheck.scala [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package controllers
```
Comment (patelh) [inline app/controllers/KafkaHealthCheck.scala]: can we put this under controllers.api

File: conf/routes [hunk 2]
```diff
@@ -6,33 +6,36 @@
...
+POST        /clusters                                         controllers.Cluster.handleAddCluster
+POST        /clusters/:c                                      controllers.Cluster.handleUpdateCluster(c:String)
+GET         /clusters/:c/createTopic                          controllers.Topic.createTopic(c:String)
+POST        /clusters/:c/topics/create                        controllers.Topic.handleCreateTopic(c:String)
+POST        /clusters/:c/topics/delete                        controllers.Topic.handleDeleteTopic(c:String, t: String)
+GET         /clusters/:c/topics/:t/addPartitions              controllers.Topic.addPartitions(c:String, t: String)
+POST        /clusters/:c/topics/:t/addPartitions              controllers.Topic.handleAddPartitions(c:String, t: String)
+GET         /clusters/:c/topics/:t/updateConfig               controllers.Topic.updateConfig(c:String, t: String)
+POST        /clusters/:c/topics/:t/updateConfig               controllers.Topic.handleUpdateConfig(c:String, t: String)
+GET         /clusters/:c/status/availableBrokers              controllers.KafkaHealthCheck.availableBrokers(c:String)
```
Comment (patelh) [inline conf/routes]: can we put all api endpoints under /api/*

File: test/controller/api/TestKafkaHealthCheck.scala [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  override val kafkaServerZkPath = broker.getZookeeperConnectionString
+  private[this] val duration = FiniteDuration(10,SECONDS)
+
+  private[this] val testClusterName = "kafka-hc-test-cluster"
+  private[this] val testTopicName = "kafka-hc-test-topic"
+
+  override protected def beforeAll() : Unit = {
+    super.beforeAll()
+    lazy val app : FakeApplication = {
+      FakeApplication()
```
Comment (patelh) [inline test/controller/api/TestKafkaHealthCheck.scala]: you need to pass in additionalConfiguration = Map("kafka-manager.zkhosts" -> kafkaServerZkPath) into FakeApplication.

**Pull request merged**